versions/2.x/models/OpenBookingError.json

```diff
@@ -1,84 +1,84 @@
 {
-    "derivedFrom": "https://schema.org/Thing", 
+    "subClassOf": "#Thing", 
```

Comment:
@thill-odi This looks to cause a lot of tests on this model and its subclasses to fail because there are now fields inherited from Thing (url, image, potentialAction, identifier, sameAs) which aren't included in InSpec.